build-scripts/lint-json.sh

```diff
@@ -0,0 +1 @@
+find . -name *json -type f -exec python -m json.tool {} >/dev/null \;
```

Comment:
This will work, but the way it's written is a bit misleading.  This redirection to `/dev/null` is actually redirecting the output of `find`, not the output of the individual commands.